core/src/data_sources/data_source.rs

```diff
@@ -638,6 +638,7 @@ impl DataSource {
                 match qdrant_client
                     .set_payload(
                         self.qdrant_collection().to_string(),
+                        None,
```

Comment:
These changes are unrelated right ?

Reply:
bump of qdrant client from 1.6 to 1.7

Reply:
Introduces a new argument (shard_key) to their interface

Reply:
got it thx !